apps/pie-microsite/src/assets/js/awesome.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable no-console */
+
+function initialiseAwesome () {
+    console.log(`
```

Comment:
Would console.info be more appropriate here? To supress warnings etc.

Reply:
I resent this comment so much hahaha - but yes you're spot on. Will update now :D